data-availability.md

```diff
@@ -11,6 +11,7 @@ Below are details regarding the historical data available for each Financial Ins
 | Longtail Institutions     | 3 months  | 3 months   |
 | Navy Federal Credit Union | 2 months  | 2 months   |
 | SunTrust                  | 18 months | 18 months  |
+| TD                        | 6  months | 12 months  |
```

Comment:
TD states officially that they provide:

1. 60 days for checking, saving and money market accounts.
2. 12 months for CD, IRA and credit cards.

My observations of actual behavior:

1. 12 months for `depository`.
2. 6 months for `credit`.

Should we publish the lesser of TD's comments and observations -- 60 days `depository`, 6 months `credit` -- to minimize support complaints?